doc/rtd/reference/base_config_reference.rst

```diff
@@ -287,6 +287,9 @@ On an Ubuntu system, :file:`/etc/cloud/cloud.cfg` should look similar to:
     # This will cause the set+update hostname module to not operate (if true)
     preserve_hostname: false
 
+    # This will create the hostname file (e.g. /etc/hostname) if it does not already exist (if true)
+    create_hostname_file: false
```

Comment:
Why add this?

> On an Ubuntu system, :file:`/etc/cloud/cloud.cfg` should look similar to:

This will only be true on GCE, right?

Reply:
Good point -- will fix!  And yes, I think it would be good to add to the modules examples for set and update hostname.  Thanks!